### PR TITLE
Re #19 - Add the nat action to tc

### PR DESCRIPTION
**Pull request by sophacles**

### CODE REVIEW

**Review by wllenyj — APPROVED** [review]

**Comment by wllenyj** [comment]
Please rebase, and solve the ci failure.

**Comment by sophacles** [comment]
Thanks for taking a look @wllenyj . I've rebased and fixed the formatting errors reported by CI. Will pay attention for additional failures in case I missed something.

**Comment by wllenyj** [comment]
@cathay4t Do you have any comments?

**Comment by cathay4t** [comment]
I have ammend the git commit message to include `Signed-off-by: Erich Heine <....>`.

File: src/rtnl/tc/constants.rs [hunk 1]
```diff
@@ -93,3 +93,12 @@ pub const TCA_EGRESS_REDIR: i32 = 1; /* packet redirect to EGRESS */
 pub const TCA_EGRESS_MIRROR: i32 = 2; /* mirror packet to EGRESS */
 pub const TCA_INGRESS_REDIR: i32 = 3; /* packet redirect to INGRESS */
 pub const TCA_INGRESS_MIRROR: i32 = 4; /* mirror packet to INGRESS */
+
+/// NAT action attr
+pub const TCA_NAT_UNSPEC: u16 = 0;
+pub const TCA_NAT_PARMS: u16 = 1;
+pub const TCA_NAT_TM: u16 = 2;
+pub const TCA_NAT_PAD: u16 = 3;
+pub const TCA_NAT_MAX: u16 = TCA_NAT_PAD;
```
Comment (cathay4t) [inline src/rtnl/tc/constants.rs:102]: Please remove `TCA_NAT_PAD` and `TCA_NAT_MAX`.
The `TCA_NAT_PAD` is unused anywhere. The `TCA_NAT_MAX` is not constant to kernel but a internal variable might change in future release of linux kernel.

**Review by cathay4t — CHANGES_REQUESTED** [review]
Besides removing `TCA_NAT_MAX` and PAD, looks good to me.

I am not TC expert, I trust you know your territory

**Comment by sophacles** [comment]
> This has been delayed for a long time. Let me amend the patch by removing TCA_NAT_MAX and PAD.

Apoligies, I lost track of that request. Thank you for taking care of  it!

**Pull request merged**